### PR TITLE
fix: evm spam test

### PR DESCRIPTION
**Pull request by jorgemmsilva**

### CODE REVIEW

File: tools/wasp-cli/go.mod [hunk 1]
```diff
@@ -163,7 +163,7 @@ require (
 	github.com/rjeczalik/notify v0.9.3 // indirect
 	github.com/rogpeppe/go-internal v1.10.0 // indirect
 	github.com/sasha-s/go-deadlock v0.3.1 // indirect
-	github.com/second-state/WasmEdge-go v0.11.2 // indirect
+	github.com/second-state/WasmEdge-go v0.12.0 // indirect
```
Comment (thethorne48) [inline tools/wasp-cli/go.mod:166]: This seems like an unrelated upgrade. Did you mean to add this?

**Review by dessaya — APPROVED** [review]
